api-gateway/src/main/java/com/petclinic/bffapigateway/domainclientlayer/CustomersServiceClient.java

```diff
@@ -1,16 +1,10 @@
 package com.petclinic.bffapigateway.domainclientlayer;
 
-import com.petclinic.bffapigateway.dtos.Login;
-import com.petclinic.bffapigateway.dtos.OwnerDetails;
+import com.petclinic.bffapigateway.dtos.*;
```

Comment:
As mentioned by peers, practicing more commits more often for our future sprints would be beneficial. 43 files to review is a bit long to review.